### PR TITLE
Add offboarding

### PR DESCRIPTION
**Pull request by NicolasMahe**

### CODE REVIEW

File: employees/README.md [hunk 1]
```diff
@@ -186,3 +186,42 @@ Make sure that when off, everything you are working on is accessible so you don'
...
+- GitHub
+- Trello
+- 1Password
+- Zeplin
+- NewRelic
+- Sendgrid
+- Facebook page
+- Linkedin
+- AirBnb
+- Spark
```
Comment (NicolasMahe) [inline employees/README.md]:
```suggestion
- Spark
- Monday
```

**Comment by antho1404** [comment]
Is there any specific reason why it's at the end? I would put it right after on-boarding

**Comment by NicolasMahe** [comment]
> Is there any specific reason why it's at the end? I would put it right after on-boarding
> 
> Also this is in the "time off / remote work"

Fixed

File: employees/README.md [hunk 2]
```diff
@@ -73,6 +73,46 @@ Create a [Github account](https://github.com/join) and watch the [handbook repos
 
 All employees get a cool MESG t-shirt and/or polo and some stickers :) YAY! 
 
+### Offboarding
```
Comment (antho1404) [inline employees/README.md]:
```suggestion
### Off-boarding
```

**Pull request merged**